Makefile

```diff
@@ -78,8 +73,12 @@ integration-tests:
 			-image $(CONTAINER_REPO):$(shell ./tools/image-tag branch)
 
 # minikube
+LIVENESS_PROBE_INITIAL_DELAY_SECONDS ?= 10
 .PHONY: minikube-install
-minikube-install: gadget-container-local
-	kubectl patch ds -n kube-system gadget -p $$'spec:\n template:\n  spec:\n   containers:\n    - name: gadget\n      image: $(CONTAINER_REPO):$(IMAGE_TAG)\n      imagePullPolicy: Never'
-	kubectl delete pod -n kube-system -l 'k8s-app=gadget'
-	kubectl get pod -n kube-system -l 'k8s-app=gadget'
+minikube-install: gadget-container
+	docker save $(CONTAINER_REPO):$(IMAGE_TAG) | pv | (eval $(shell $(MINIKUBE) -p minikube docker-env | grep =) && docker load)
```

Comment:
What about using minikube cache -> https://minikube.sigs.k8s.io/docs/handbook/pushing/#2-push-images-using-cache-command? It has the advantage that is compatible with any container runtime, even if it's a bit slower. 
`minikube cache add $(CONTAINER_REPO):$(IMAGE_TAG) `

Reply:
That's cool, I didn't know about it

Reply:
Btw, the GitHub Action could make use of this to address the TODO.
```
# TODO: how to avoid pushing a container before running integration tests
```